src/Events/Editor/Full_Site/Assets.php

```diff
@@ -0,0 +1,59 @@
+<?php
```

Comment:
Coding standards would require a file header; not required as we do not consistently add it in the code, but a note.